Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM praekeltfoundation/vumi
+MAINTAINER Praekelt Foundation <dev@praekeltfoundation.org>
+
+# Install nodejs 0.10 from the Debian repo & make executable available as 'node'
+RUN apt-get-install.sh nodejs && \
+    update-alternatives --install /usr/bin/node node /usr/bin/nodejs 50
+
+ENV VXSANDBOX_VERSION "0.6.0"
```

Comment:
We might want to update `utils/bump-version.sh` to update this version number and the one in `node_4.x.dockerfile`?